### PR TITLE
Add ARIA Code Hints

### PR DESCRIPTION
**Pull request by Coder206**
With the aria-atomic, aria-activedescendant and aria-flowto. Has been compared to WYSIWYG for ARIA, its a match. @RaymondLim Plus added "none" to aria-autocomplete

### CODE REVIEW

**Comment by ingorichter** [comment]
Great addition. Thank you. Would you mind changing the formatting back to the previous style? BTW: did you sign the contributor license agreement (CLA)?

**Comment by Coder206** [comment]
@ingorichter I am not sure what you mean by "previous style". Yes I think I did!

**Comment by ingorichter** [comment]
Looking at the diff, you can see that the previous formatting was different. Every entry was in one line.

**Comment by Coder206** [comment]
@ingorichter Ok!

**Comment by Coder206** [comment]
@ingorichter I can't tell if this is an error on my part. I don't think changing a format can fail a build...

**Comment by Coder206** [comment]
@ingorichter Is my work done here?

**Comment by twang2218** [comment]
@Coder206 The code style are different. Many irrelevant code have been touched in the first commit. Here is the screenshot.



Did you turn on the code beautify/format? Maybe you can turn them off and restore the original format, and try commit again. Thanks.

File: src/extensions/default/HTMLCodeHints/HtmlAttributes.json [hunk 1]
```diff
@@ -1,227 +1,261 @@
...
-                                             "ia", "ie", "iu", "ik", "ga", "it", "ja", "jv", "kn", "ks", "kk", "rw", "ky", "rn", "ko", "ku", "lo", "la", 
-                                             "lv", "li", "ln", "lt", "mk", "mg", "ms", "ml", "mt", "mi", "mr", "mo", "mn", "na", "ne", "no", "oc", "or", 
-                                             "om", "ps", "pl", "pt", "pa", "qu", "rm", "ro", "ru", "sz", "sm", "sg", "sa", "sr", "sh", "st", "tn", "sn", 
-                                             "ii", "sd", "si", "ss", "sk", "sl", "so", "es", "su", "sw", "sv", "tl", "tg", "ta", "tt", "te", "th", "bo", 
-                                             "ti", "to", "ts", "tr", "tk", "tw", "ug", "uk", "ur", "uz", "vi", "vo", "wa", "cy", "wo", "xh", "yi", "yo", 
-                                             "zu"], 
-                            "global": "true" },
-    "role":               { "attribOption": ["alert", "alertdialog", "article", "application", "banner", "button", "checkbox", "columnheader", "combobox", 
+    "accesskey":                { "attribOption": [], "global": "true" },
```
Comment (Coder206) [inline src/extensions/default/HTMLCodeHints/HtmlAttributes.json]: @twang2218 I thought I did... I am not sure what is going on.

**Comment by twang2218** [comment]
@Coder206 I send a pull request in your repo to you to fix the format issue some days ago, could you merge it, so this issue can be fixed?

**Comment by Coder206** [comment]
@twang2218 Ok sounds good.

**Comment by ficristo** [comment]
@twang2218 you need to sign the CLA http://dev.brackets.io/brackets-contributor-license-agreement.html
While indirectly, you still have modified the source.

@ingorichter do you have time to review this?

**Comment by twang2218** [comment]
@ficristo ok. It's done.

**Comment by Coder206** [comment]
@ficristo I am quite sure I have filled it out. What makes you think I didn't?

**Comment by ficristo** [comment]
@Coder206 Could you solve the merge conflicts?

**Comment by Coder206** [comment]
@ficristo I am very confused as to what I have to do know. I have a enabled edits from maintainers.

**Comment by ficristo** [comment]
On your local branch you should merge master in it, solve eventual conflicts and push again your changes.

**Comment by Coder206** [comment]
@ficristo Ok I am wondering why this PR wasn't merged beforehand.

**Comment by Coder206** [comment]
Ping!

**Comment by ficristo** [comment]
If your ping is about this not be merged, keep in mind that at the moment Adobe is focused on Dreamweaver so there is much help there.

Personally I was thinking @ingorichter would have reviewed this.
Then as some weeks ago, I was hoping a 1.8 release would come out, and I was waiting it before reviewing your PR myself.

For the conflicts they happen: you should resolve them.

**Comment by Coder206** [comment]
@ficristo Done!

**Review by zaggino — CHANGES_REQUESTED** [review]
You need to fix the diff, you merged master incorrectly removing `autocomplete` options: 
image

**Comment by Coder206** [comment]
Changes made :)

**Comment by zaggino** [comment]
:+1: thanks @Coder206

**Comment by ingorichter** [comment]
Sorry. I somehow missed that

**Pull request merged**